nighthawk/test/certs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Can you reuse the certs that already exist in https://github.com/envoyproxy/envoy/tree/master/test/config/integration/certs?

Reply:
Yes, indeed. I symlinked the certs we need, and after tweaking the environment variables, I was able to drop the script that generates them and use those instead.